frontend/src/pages/workspace/[slug]/layout.js

```diff
@@ -322,7 +314,7 @@ const NestedLayout = ({ children }) => {
                 {step?.substeps?.length > 0 && (
                   <div
                     className="substeps"
-                    style={{ height: 52 * step.substeps.length }}
+                    style={{ height: 60 * step.substeps.length }}
```

Comment:
Why have you changed this? The height now does not show correctly from what I see.